package.json

```diff
@@ -185,6 +185,7 @@
         "lint-staged": "15.2.10",
         "mockdate": "3.0.5",
         "msw": "2.4.3",
+        "node-network-devtools": "^1.0.22",
```

Comment:
Please pin the dep to its exact version and rebuild pnpm-lock

Reply:
fixed